### ISSUE TITLE
Expected masked_lm_accuracy

### ISSUE THREAD

**Issue by okgrammer** [google-research/bert#90]
The paper stated that the model was pretrained for 1M steps. May I know how high roughly the masked_lm_accuracy is expected to be at the end of training? Is a development set used in paper?

**Comment by IntelOSt** [comment]
https://multimediashop.net

**Comment by jacobdevlin-google** [comment]
I don't know the exact accuracy, but the held-out natural log likelihood for `BERT-Base` was around -1.4 (so e^1.4 = 4.0 which was the perplexity). It depends on the language and corpus but you should expect something better than -2.0 (better than == closer to zero).

**Comment by guotong1988** [comment]
Same question, thank you.

**Comment by loveJasmine** [comment]
in the readme 👍 

***** Eval results *****
  global_step = 20
  loss = 0.0979674
  masked_lm_accuracy = 0.985479
  masked_lm_loss = 0.0979328
  next_sentence_accuracy = 1.0
  next_sentence_loss = 3.45724e-05
